### PR TITLE
Upgraded to support VS2017

### PR DESCRIPTION
**Pull request by justcla**
Can build and run in both VS2015 and VS2017

### CODE REVIEW

File: EasyMotion/EasyMotion.csproj [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
+  <Import Project="..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.12-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props" Condition="'$(VisualStudioVersion)' == '14.0' And Exists('..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.12-pre\build\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.props')" />
```
Comment (jaredpar) [inline EasyMotion/EasyMotion.csproj]: I think we can just delete VS 14.0 support altogether at build time.  I've been settling on the idea of limting activey F5 development to VS 2017.

Comment (justcla) [inline EasyMotion/EasyMotion.csproj]: We could. But we don't need to. I very carefully implemented full VS2015/VS2017 round-tripping. This is the way we are encouraging all extension developers to build their VSIXes.

**Comment by justcla** [comment]
Can we bring back VS2012/VS2013 run-time support?

**Comment by jaredpar** [comment]
> Can we bring back VS2012/VS2013 run-time support?

Sure.  Should be as simple as:

1. Changing the version of the WPF editor references to 12.0.0.0.
1. Adding them as an installation target

I'm not sure why I ever changed 1 above.

**Pull request merged**